hazelcast/src/main/java/com/hazelcast/internal/diagnostics/HealthMonitor.java

```diff
@@ -353,9 +357,9 @@ private void renderProxy() {
 
         private void renderLoad() {
             sb.append("load.process").append('=')
-                    .append(format("%.2f", osProcessCpuLoad.read())).append("%, ");
+                    .append(format("%.2f", osProcessCpuLoad.read() * PERCENTAGE_MULTIPLIER)).append("%, ");
             sb.append("load.system").append('=')
-                    .append(format("%.2f", osSystemCpuLoad.read())).append("%, ");
+                    .append(format("%.2f", osSystemCpuLoad.read() * PERCENTAGE_MULTIPLIER)).append("%, ");
```

Comment:
FYI: we previously would scale the 0.0-1.0 range as returned by the MX bean to 0-100 using a probe function. Now this is read directly from the MX bean so we have to scale here instead.